profiles/common/modules/features/nexteuropa_core/nexteuropa_core.tokens.inc

```diff
@@ -27,13 +27,12 @@ function nexteuropa_core_tokens($type, $tokens, array $data = array(), array $op
     $node = $data['node'];
     foreach ($tokens as $name => $original) {
       if (strstr($name, ':parents-uri') !== FALSE) {
-        list($field_name, $delta,) = explode(':', $name);
+        list($field_name) = explode(':', $name);
         $field_name = str_replace('-', '_', $field_name);
         if (isset($node->{$field_name})) {
           $node_wrapper = entity_metadata_wrapper('node', $node);
-          $terms = $node_wrapper->{$field_name}->value();
-          if (isset($terms[$delta - 1])) {
-            $term = $terms[$delta - 1];
+          $term = $node_wrapper->{$field_name}->value();
```

Comment:
Hi Gilles,
I just made some tests and tokens already has validation to prevent the use of tokens in incorrect contexts. In this case, if someone tries to force this token with a multivalue term reference, he won't be able to. See token_element_validate inside the token module.
Based on this, I don't think we should implement extra validation steps in our code.